### ISSUE TITLE
[Feature Request][Legacy UI] Tweak frontline attack ground unit generation groupings to be tighter

### ISSUE THREAD

**Issue by phntmknight321** [shdwp/dcs_liberation#100]
This is based on legacy UI behavior, but the current way it generates units for frontline attack missions sometimes feels a bit unrealistic. This is entirely subjective, but I am personally not a fan of the huge line of enemy tanks approaching all parallel to each other. 

I recognize this may be a scripting challenge, but wanted to throw my .02 in. Please delete if is not appropriate here.

**Comment by wrycu** [comment]
I think this is being addressed in the QT rewrite.  @Khopa can you comment?

**Comment by Khopa** [comment]
Yes, i had some ideas for something a bit more complex in qt_ui branch.

But this is far from being done (or even started 😁 ), as i've not really worked on the project lately... 😕 

***

I wanted to make it so the frontline would be made of multiple lines, a bit like in this picture i made back when we were discussing the air defense rework : 

![image](https://user-images.githubusercontent.com/2546901/81485657-22138e00-924f-11ea-90be-b57b42f873ca.png)

Basically, heavy units such as tanks would be first in line, then ATGM and lighter units, then artillery units and close range AA in the back, and finally maybe even a logistic supply line (which does not appear in the picture).

Regarding these ground units behaviour, first thing i wanted to do is to have the player select either a "defensive" or "offensive" stance for them before the mission start, in order to decide whether they would try to attack the ennemy or not. And the AI would basically choose "offensive" stance when it has more tanks, or else "defensive". And only the "armored" units would try to rush in.

Then maybe after that would be done, a more complex behaviour than "let's rush forward !" for the ground units could have been implemented, but I did not thought too much about it.

This is not an easy task, and we'd have to be careful not to cause too much lag with the ground units behaviour. (Making ground units move in DCS is usually a big source of lag for most mission, especially when multiplayer is involved (or at least, it used to be).) 

***

But that's just some ideas i had, i did not implement them.
